.eslintrc

```diff
@@ -1,15 +1,15 @@
 {
   "parser": "babel-eslint",
-  "extends": "eslint:recommended",
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended"
```

Comment:
I ran the linter before I made the PR and everything was passing with `plugin:react/recommended`, so I saw no harm in enabling it.